index.html

```diff
@@ -3,9 +3,45 @@
   <head>
     <title>Relaxr</title>
     <link rel="stylesheet" type="text/css" href="css/style.css">
+    <link rel="stylesheet" type="text/css" href="css/normalize.min.css">
+    <link href='https://fonts.googleapis.com/css?family=Open+Sans' rel='stylesheet' type='text/css'>
   </head>
+
   <body>
-    <header>
-    </header>
+  <header>
+    <h1>Relaxr</h1>
+    <h2>Get piece of mind with a single tap</h2>
+    <h3>Get it Now</h3>
+  </header>
+
+   <article>
+   <h4>Benefits</h4>
+   <p>The perfect personal assistant. Relaxr does your job for you so you can enjoy life as it is meant to be.
+   <ul>
+   <li>Schedule meetings for you</li>
+   <li>Excel automation to complete your work for you</li>
+   <li>Responds to emails on your behalf</li>
+   <li>Does all your work for you with our revolutionary AI technology</li>
+   </ul>
+   </article>
+
+   <section>
+   <div id="p1">"Relaxr changed my life. I've been able to travel the world, spend limited time working and my boss keeps thanking me for crushing work.</div>
+   <div id="p2">- Amanda, Intuit</div>
+   </section>
+
+   <aside>
+   <h5>Sign Up Now!</h5>
+   </aside>
+
+   <footer>
+   <div id="logo">Relaxr</div>
+   
+   <img src="../images/twitter.png" alt="Twitter Logo">
```

Comment:
Remove the `../` from your images and they will work. You do not need these because you do not need to traverse up the file tree. If you were applying these images via a css background-image you would need to do this.